### ISSUE TITLE
Single input date component - label mask screen reader issue in Chrome/JAWS

### ISSUE THREAD

**Issue by malagapp** [CMSgov/design-system#2297]
**Describe the bug**
In Chrome browser using JAWS the screen reader does not announce the label mask correctly for the single input date component. It only says MM/DD/YYY. It announces only 3 Y's.

**To Reproduce**
1. Go to 'https://design.cms.gov/components/date-picker'
2. Start JAWS screen reader
3. Go to the date field
4. JAWS announces MM/DD/YYY twice with only 3 Y's for the label mask for the date field.

**Expected behavior**
JAWs in chrome should announce the correct format MM/DD/YYYY. 

**Screenshots**
No screenshots

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version [108.0.5359.99]

**Smartphone (please complete the following information):**
N/A

**Additional context**
It works correctly  in Firefox browser using NVDA screen reader but does not work in Chrome browser with JAWS.

**Comment by SparrowSpit** [comment]
@malagapp see [this note on Freedom Scientific's Github](https://github.com/FreedomScientific/VFO-standards-support/issues/636). Apparently this is a settings issue within JAWS. In the Github Issue convo, there are directions on how to adjust the settings to allow for more repeated characters to be read.

**Comment by Karak888** [comment]
@SparrowSpit has added that the label mask screed reader issue is a settings issue within JAWS. Will add directions Dan noted for how to adjust the settings to allow for more repeated characters to be read.  Closing this issue
